hibernate-core/src/test/java/org/hibernate/orm/test/annotations/xml/ejb3/NonExistentOrmVersionTest.java

```diff
@@ -30,7 +29,5 @@ public void testNonExistentOrmVersion() {
 		}
 		catch ( InvalidMappingException expected ) {
 		}
-		catch ( UnsupportedOrmXsdVersionException expected ) {
```

Comment:
Well, I dunno, it's confusing: apparently it throws *neither of them*, because the test passes without this `catch` block ... unless I misunderstood something?

Reply:
:man_shrugging: so then its fine I guess.